site/src/components/segmented-controls.md

```diff
@@ -0,0 +1,395 @@
+			<p>Tab Item 6</p>
+		</div>
+	</div>
+
+	<!-- After Tabpanels -->
+	<div class="segmented-controls-footer border-top buttons padding-top-sm">
+		<button class="button button-sm button-primary" data-segmented-control-nav="previous">
+			<span class="icon" aria-hidden="true">
+				<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 448">
+					<path
+						class="fill-current-color"
+						d="M448 224H62.625l148.688 148.688-22.625 22.625L1.375 208 188.688 20.688l22.625 22.625L62.625 192H448v32z"
+					/>
+				</svg>
+			</span>
+			<span>Previous</span>
+		</button>
+
+		<button class="button button-sm button-primary button-filled" data-segmented-control-nav="next">
+			<span>Next</span>
+			<span class="icon" aria-hidden="true">
+				<svg xmlns="http://www.w3.org/2000/svg" viewBox="-255 57 448 448">
+					<path
+						class="fill-current-color"
+						d="M-253.625 281v-32H131.75L-16.938 100.313 5.687 77.688 193 265 5.687 452.313l-22.625-22.625L131.75 281h-385.375z"
+					/>
+				</svg>
+			</span>
+		</button>
+	</div>
+</tab-container>
+```
+
+## Size modification
+
+Using the `segmented-controls-lg` can be used to make segmented control bigger than the default.
+
+```html
+<tab-container data-segmented-controls-container>
+	<div class="segmented-controls segmented-controls-lg" slot="tablist-wrapper">
+		<button
+			type="button"
+			class="segmented-control-previous margin-right-xxs"
+			title="Previous segment"
+			data-segmented-control-nav="previous"
+		>
+			<span class="icon" aria-hidden="true">
+				<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 448">
+					<path
+						class="fill-current-color"
+						d="M448 224H62.625l148.688 148.688-22.625 22.625L1.375 208 188.688 20.688l22.625 22.625L62.625 192H448v32z"
+					/>
+				</svg>
+			</span>
+		</button>
+		<div class="segmented-control-list" role="tablist" aria-orientation="horizontal">
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-two-1"
+				role="tab"
+				aria-selected="true"
+				data-segmented-control
+			>
+				Define Strategy
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-two-2"
+				role="tab"
+				data-segmented-control
+			>
+				Plan
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-two-3"
+				role="tab"
+				data-segmented-control
+			>
+				Prepare
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-two-4"
+				role="tab"
+				data-segmented-control
+			>
+				Adopt
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-two-5"
+				role="tab"
+				data-segmented-control
+			>
+				Govern
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-two-6"
+				role="tab"
+				data-segmented-control
+			>
+				Manage
+			</button>
+		</div>
+		<button
+			type="button"
+			class="segmented-control-next margin-left-xxs"
```

Comment:
Don't think our examples/component should have margin atomics. let's handle this within the component itself rather than with atomics - same for the previous button: `segmented-control-previous margin-right-xxs`. Would be good to tighten up these gaps on mobile anyway:

<img width="166" alt="image" src="https://github.com/microsoft/atlas-design/assets/30843002/aa9ae670-342c-41d1-b99b-267b6428e772">

Reply:
Looks like I missed some removals.  Ill check it out

Reply:
This should be adjusted now

Reply:
Removed the margin, but acknowledged that the space needs some magic in future.  Magic to include some css logic to reduce next/prev?  Will seek feedback on options here.